<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #181b24;</string>
  </property>
  <widget class="QLabel" name="com_port_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>COM Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="temperature_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="com_port_text">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 5px;
color: white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="temperature_text">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 5px;
color: white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>150</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 5px;
color: white;</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
